proto/penumbra/penumbra/view/v1alpha1/view.proto

```diff
@@ -128,8 +128,8 @@ message TransactionPlannerRequest {
     // The memo for the requested TransactionPlan.
     // The memo must be unspecified unless `outputs` is nonempty.
     core.transaction.v1alpha1.MemoPlaintext memo = 3;
-    // Identifies the account group to query.
-    optional core.crypto.v1alpha1.AccountGroupId account_group_id = 14;
+    // Identifies the account group to query. Defaults to 0.
+    core.crypto.v1alpha1.AccountGroupId account_group_id = 14;
```

Comment:
0 isn't a valid account group ID, right? So this comment should read
```
// If present, identifies the account group to query.
```
?

Reply:
Or we could just drop these entirely, since we don't use them at all and have no concrete plans to do multi-account view servers.